### PR TITLE
Add functionality to toggle the star image for favoriting and unfavor…

### PR DESCRIPTION
**Pull request by lschnell8**
PR Template: 

Is this a fix or a feature?
[] Bug fix
[x] Feature
[] Refactor

Describe the feature - fix - refactor:
Created logic to toggle the favoriting feature for the idea cards. I reassigned the image sources and ids on the click event to align with the image that needs to be displayed. The function also reassigns the star property in the idea class to true or false depending on which image is displayed.

How was this tested?
[x] open index.html
[] localhost 
[] terminal

Where should the reviewer start - test:
See toggleFavoriteStatus function starting on line 87 in main.js

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -86,14 +85,16 @@ function deleteCard() {
 };
 
 function toggleFavoriteStatus() {
+  var starItem = event.target
```
Comment (ninabrissey) [inline main.js:92]: Thank you for creating this new variable. Makes it so much cleaner.

**Review by csmucker83 — APPROVED** [review]
Improved star toggle switch. Completed iteration 3.

File: main.js (hunk 1 above)
Comment (ninabrissey) [reply main.js:92]: add semi-colon at end of line 88

File: main.js [hunk 2]
```diff
@@ -86,14 +85,16 @@ function deleteCard() {
...
   for (var i = 0; i < ideas.length; i++) {
     if(event.target.id === 'inactiveStar' && ideas[i].id === parseInt(event.target.closest('article').id)) {
       ideas[i].star = true;
-      // emptyStar.classList.add('hidden');
-      // filledStar.classList.remove('hidden');
-      // if(img.className === 'star') {
-      //
-      }
+      starItem.src = 'assets/star-active.svg'
```
Comment (ninabrissey) [inline main.js:96]: Great job on this. So exciting.

**Review by ninabrissey — APPROVED** [review]

**Pull request merged**